packages/dds/sequence/src/sequence.ts

```diff
@@ -680,11 +694,15 @@ export abstract class SharedSegmentSequence<T extends ISegment>
 
     private initializeIntervalCollections() {
         // Listen and initialize new SharedIntervalCollections
-        this.intervalMapKernel.eventEmitter.on("valueChanged", (ev: IValueChanged) => {
+        this.intervalMapKernel.eventEmitter.on("create", (ev: IValueChanged, local: boolean) => {
             const intervalCollection = this.intervalMapKernel.get<IntervalCollection<SequenceInterval>>(ev.key);
             if (!intervalCollection.attached) {
                 intervalCollection.attachGraph(this.client, ev.key);
             }
+            if (ev.previousValue === undefined) {
```

Comment:
do we still need this with the new create event?

Reply:
Do we need to check if ev.previousValue === undefined? No, probably not.